core/domain/summary_services.py

```diff
@@ -60,6 +60,22 @@
 
 
 def get_human_readable_contributors_summary(contributors_summary):
+	"""Get contributors summary in human readable form.
+	
+	Args:
+		contributors_summary: A dict of contributors; the keys are users id and 
```

Comment:
Throughout, please use spaces not tabs, and follow existing codebase style. (Did the pre-commit lint checks not catch this?)

Also, follow format of completed files in #2394 as closely as possible. E.g.:

```
    Args:
        contributors_summary: dict. The keys are user_ids, and the corresponding values
            are the number of commits each user has made.
```

Note the explicit specification of the arg type.